src/manifest/modules/dbtProjectLog.ts

```diff
@@ -112,6 +115,9 @@ export class DBTProjectLog implements Disposable {
     if (this.outputChannel !== undefined) {
       this.outputChannel.dispose();
     }
+    if (this.logFileWatcher) {
+      this.logFileWatcher.dispose();
```

Comment:
This will break the logger. The path to the logs is not changing. only the name of the outputchannel and project name is changing.